docs/contribute/release_checklist.md

```diff
@@ -22,110 +64,73 @@ These are instructions for preparing a release branch for Firefox Android and st
-   - Set the `channel` to `GeckoChannel.BETA`.
-
-    ```diff
-    diff --git a/android-components/plugins/dependencies/src/main/java/Gecko.kt b/android-components/plugins/dependencies/src/main/java/Gecko.kt
-    --- a/android-components/plugins/dependencies/src/main/java/Gecko.kt
-    +++ b/android-components/plugins/dependencies/src/main/java/Gecko.kt
-    @@ -14,7 +14,7 @@ object Gecko {
-         /**
-          * GeckoView channel
-          */
-    -    val channel = GeckoChannel.NIGHTLY
-    +    val channel = GeckoChannel.BETA
-     }
-
-     /**
-    ```
-7. Create a commit named `Switch to GeckoView Beta` for this change. This change can either be directly committed to the `releases_v[beta_version]` branch or a pull request can be created against it and then merged. Once landed, it is expected that this change will temporarily break builds on the branch. The next step will fix them.
-8. The Github Action will automatically bump the GeckoView version once the new Beta build is created and uploaded to Maven, as shown in [#324](https://github.com/mozilla-mobile/firefox-android/pull/324) for example.
-9. In [ApplicationServices.kt](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/plugins/dependencies/src/main/java/ApplicationServices.kt):
-  - Set `CHANNEL` to `ApplicationServicesChannel.RELEASE`
-  - Set `VERSION` to `[major-version].0`
-
-```diff
-diff --git a/android-components/plugins/dependencies/src/main/java/ApplicationServices.kt b/android-components/plugins/dependencies/src/main/java/ApplicationServices.kt
-index f64b26d7a1..72574ab057 100644
---- a/android-components/plugins/dependencies/src/main/java/ApplicationServices.kt
-+++ b/android-components/plugins/dependencies/src/main/java/ApplicationServices.kt
-@@ -3,8 +3,8 @@
-  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
--// These lines are generated by android-components/automation/application-services-nightly-bump.py
--val VERSION = "114.20230420185101"
--val CHANNEL = ApplicationServicesChannel.NIGHTLY
-+val VERSION = "114.0"
-+val CHANNEL = ApplicationServicesChannel.RELEASE
-```
-10. Create a commit named `Swift to Application Services Release`.  (Note: application-services releases directly after the nightly cycle, there's no beta cycle).
-11. Once all three of the above commits have landed, create a new `Firefox Android (Android-Components, Fenix, Focus)` release in [Ship-It](https://shipit.mozilla-releng.net/) and continue with the release checklist per normal practice.
+4. Once all three of the above commits have landed, create a new `Firefox Android (Android-Components, Fenix, Focus)` release in [Ship-It](https://shipit.mozilla-releng.net/) and continue with the release checklist per normal practice.
+5. The Github Action will automatically bump the GeckoView version once the new Beta build is created and uploaded to Maven, as shown in [#324](https://github.com/mozilla-mobile/firefox-android/pull/324) for example.
```

Comment:
Swap the content in Step 4 and  Step 5
The final step is setting up the build in ShipIt after the automatic GeckoView version bump PR is merged.
"Once all four of the above commits have landed..."